monkestation/code/modules/clothing/suits/coats.dm

```diff
@@ -104,6 +104,13 @@
 	icon_state = "guardman_vest"
 
 
+/obj/item/clothing/suit/armor/blueshield
+	name = "blueshield coat"
+	desc = "A stylish kevlar coat worn by the blueshield agent."
+	icon = 'monkestation/icons/obj/clothing/suits.dmi'
+	worn_icon = 'monkestation/icons/mob/clothing/suit.dmi'
+	icon_state = "blueshield"
+
```

Comment:
There doesn't appear to be an armor value datum for this?

Reply:
Should inherit from above